src/nft/ZoraCreator1155Impl.sol

```diff
@@ -623,6 +698,25 @@ contract ZoraCreator1155Impl is
         }
     }
 
+    /// @notice Withdraws ETH from the Zora Rewards contract
+    function withdrawRewards(uint256 amount) public onlyAdminOrRole(CONTRACT_BASE_ID, PERMISSION_BIT_FUNDS_MANAGER) {
```

Comment:
is having an `amount` necessary? how about just withdrawing the full balance? it could make things simpler